<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QWidget" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="mainmenu">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>10</y>
       <width>751</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Тестирование&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>230</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QLabel" name="test_intro_label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>671</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;по теме &amp;quot;Движение заряженной частицы в электрическом поле&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="questions_count_label">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>210</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Количество вопросов:</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>269</x>
       <y>180</y>
       <width>271</width>
       <height>411</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="menu">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Начать тестирование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_results">
        <property name="minimumSize">
         <size>
          <width>37</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Посмотреть статистику</string>
        </property>
        <property name="iconSize">
         <size>
          <width>29</width>
          <height>28</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="return_to_menu_button_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Вернуться в главное меню</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>582</x>
       <y>280</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>anonymous</string>
     </property>
    </widget>
    <widget class="QLabel" name="username_label">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя пользователя:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>160</y>
       <width>801</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(83, 83, 83);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Меню тестирования&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="test">
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_question_type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>306</y>
       <width>781</width>
       <height>251</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="closed_question">
      <widget class="QCheckBox" name="Answer1_checkBox">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>40</y>
         <width>72</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="Answer1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(144, 203, 255);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Answer1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLabel" name="Answer2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(144, 203, 255);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Answer2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLabel" name="Answer3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(144, 203, 255);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Answer3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>20</number>
       </property>
      </widget>
      <widget class="QLabel" name="Answer4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>651</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(144, 203, 255);
color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Answer4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>20</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="Answer2_checkBox">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>100</y>
         <width>72</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="Answer3_checkBox">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>160</y>
         <width>72</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="Answer4_checkBox">
       <property name="geometry">
        <rect>
         <x>710</x>
         <y>220</y>
         <width>72</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="open_question">
      <widget class="QLineEdit" name="answer">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>641</width>
         <height>111</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="youranswer">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ваш ответ:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="prev">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>570</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>570</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="finish">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>570</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Завершить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="result">
    <widget class="QLabel" name="label_result">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>20</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 107, 1);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Результаты тестирования</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_result">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>751</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_your_answers">
     <property name="geometry">
      <rect>
       <x>186</x>
       <y>100</y>
       <width>421</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 107, 1);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Ваши ответы</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_proc_right_answers">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>510</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 107, 1);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Процент правильных ответов:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_your_answers_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 107, 1);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Ваша оценка:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_proc">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>510</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 107, 1);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_ocenka">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>550</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 107, 1);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="return_to_menu_button">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>560</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Вернуться в главное меню</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="stats">
    <widget class="QTextBrowser" name="textBrowser_stats">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>781</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="return_to_menu_button_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>570</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Вернуться в главное меню</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
